script.js

```diff
@@ -50,4 +50,393 @@ menuIcon.addEventListener('keydown', (e) => {
+}
+
+for (let i = 1; i < 2; i += 1) {
+  const popupBackground = document.querySelector('.popup_background');
+  const popupWindow = document.querySelector('.popup');
+
+  const closeBtn = document.createElement('div');
+  closeBtn.classList.add('close_buttons');
+
+  const popupCloseMobile = document.createElement('img');
+  popupCloseMobile.src = projectInfo[i].closeBtnMobile;
+  popupCloseMobile.classList.add('close_mobile');
+
+  const popupCloseDesktop = document.createElement('img');
+  popupCloseDesktop.src = projectInfo[i].closeBtnDesktop;
+  popupCloseDesktop.classList.add('close_desktop');
+
+  closeBtn.append(
+    popupCloseMobile,
+    popupCloseDesktop,
+  );
+
+  const popupImg = document.createElement('div');
+  popupImg.classList.add('popup_images');
+
+  const popupMobileImage = document.createElement('img');
+  popupMobileImage.src = projectInfo[i].popupImageMobile;
+  popupMobileImage.classList.add('mobile_popup_image');
+
+  const popupDesktopImage = document.createElement('img');
+  popupDesktopImage.src = projectInfo[i].popupImageDesktop;
+  popupDesktopImage.classList.add('desktop_popup_image');
+
+  popupImg.append(
+    popupMobileImage,
+    popupDesktopImage,
+  );
+
+  const containerForHeaderContainerAndPopupButtons = document.createElement('div');
+  containerForHeaderContainerAndPopupButtons.classList.add('container_for_header_container_and_popup_buttons');
+
+  const headerContainer = document.createElement('div');
+  headerContainer.classList.add('header_container');
+
+  const popupHeader = document.createElement('h3');
+  popupHeader.classList.add('popup_header');
+  popupHeader.innerHTML = projectInfo[i].popupCardHeader;
+
+  headerContainer.appendChild(
+    popupHeader,
+  );
+
+  const popupButtonsDesktop = document.createElement('div');
+  popupButtonsDesktop.classList.add('popup_buttons_desktop');
+
+  const seeSourceBtn = document.createElement('button');
+  seeSourceBtn.type = 'button';
+  seeSourceBtn.classList.add('see-project');
+  seeSourceBtn.innerHTML = `
+        See Source
+          <span><img src="assets/popup/icons/see-source-icon.svg"></span>
+    `;
+
+  const seeLiveBtn = document.createElement('button');
+  seeLiveBtn.type = 'button';
+  seeLiveBtn.classList.add('see-project');
+  seeLiveBtn.innerHTML = `
+        See Live
+          <span><img src="assets/popup/icons/see-live-icon.svg"></span>
+    `;
+
+  const seeLiveBtnDesktop = document.createElement('button');
+  seeLiveBtnDesktop.type = 'button';
+  seeLiveBtnDesktop.classList.add('see-project');
+  seeLiveBtnDesktop.innerHTML = `
+        See Live
+          <span><img src="assets/popup/icons/see-live-icon.svg"></span>
+    `;
+
+  const seeSourceBtnDesktop = document.createElement('button');
+  seeSourceBtnDesktop.type = 'button';
+  seeSourceBtnDesktop.classList.add('see-project');
+  seeSourceBtnDesktop.innerHTML = `
+        See Source
+          <span><img src="assets/popup/icons/see-source-icon.svg"></span>
+    `;
```

Comment:
- [ ] Good job working on your popup. Right now, when I click on any see project, the popup shows correctly. However, when you look keenly, you notice that somehow the popup is being centered but not at the page view port widths and heights but at the whole length of the webpage. This means that on mobile, I click on the popup then scroll down to see the popup. This is not a very effective way of having it. 

**To fix this,** Make the popup centered by the user's screen viewport height/ width and not the whole website's length.


**What I mean on mobile view:**

[Screencast from 17-11-2022 03:05:28 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/202441779-8748bceb-1f8c-4702-894e-0370bb7ab7ac.webm)

**Now what I mean on the desktop view**

[Screencast from 17-11-2022 03:06:24 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/202442082-5a263ff2-ec3b-4268-a472-a492069672f3.webm)


**Hope now** you understand what I mean and will address it.

- [ ] Your design on mobile view has a much larger marins/ padding on the background. This then makes the other sections to get "squeezed" and distorts the overall design. Kindly make this change

**This is how your mobile version currently looks like:**

[Screencast from 17-11-2022 03:27:20 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/202446271-f001c9d1-9a57-4b2b-8a47-32ceba0c4c8c.webm)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/202446403-726f0c0b-ef5b-4927-bb28-07eb5bc0fac5.png)